sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/CertificateClientLiveTests.cs

```diff
@@ -557,14 +558,175 @@ public async Task ValidateMergeCertificate()
             CollectionAssert.AreEqual(mergedServerCertificate.Cer, completedServerCertificate.Cer);
         }
 
-        // Backup Restore
-        // GetCertificates
-        // GetCertificatsIncludePending
-        // GetCertificateVersions
-        // GetDeletedCertificates
-        // GetUpdatePolicy
-        // IssuerCrud
-        // ContactsCrud
+        [Test]
+        public async Task VerifyGetIssuer()
+        {
+            string issuerName = Recording.GenerateId();
+
+            string providerName = "SSLAdmin";
+
+            CertificateIssuer issuer = new CertificateIssuer(issuerName, providerName);
+
+            RegisterForCleanupIssuer(issuerName);
+
+            await Client.CreateIssuerAsync(issuer);
+
+            CertificateIssuer getIssuer = await Client.GetIssuerAsync(issuerName);
+
```

Comment:
### Issue1

**Descrption:**
Method GetIssuer() throw a exception while try to get issuer pre created by portal.

**Reproduce Steps:**

1. Create CertificateIssuer by portal.
Navigate to Certificate -> Certificate Authorties, create a issuer named "test" with provider "SSLAdmin"

2. Get the issuer by code.
Exception throw: System.InvalidOperationException: 'The requested operation requires an element of type 'String', but the target element has type 'Array'.'

Reply:
A good find, but not blocking. Please open a separate issue assigned to me and I'll have a look. For now, you should be able to proceed since this test code needs to be repeatable.